### ISSUE TITLE
How to deal with patient change

### ISSUE THREAD

**Issue by bvdh** [cds-hooks/docs#97]
The EHR typically views the data of a single patient.
If a link on a card is triggered, a second browser session on the same patient may be started.

What happens when the EHR changes patient? Shouldn't we include a message that allows the EHR to indicate that the current patient session has stopped? Or is this covered by the SMART app launch URL specification?

Bas

**Comment by isaacvetter** [comment]
Hey @bvdh ,

Your question is: How do we prevent an EHR and SMART app from becoming "out-of-sync"?

This is definitely outside the scope of the CDS Hooks spec. Also note that the SMART spec doesn't address this either. 

In practice, the best (and I think, common) approach to this problem is by the EHR / SMART Server hosting the app within an iframe within the patient's chart. By doing this, the SMART app is naturally subject to changes made to its container. 

To illustrate: the patient's chart is a web page in a browser, the SMART app is an iframe within that web page, when the user changes the patient, the SMART app iframe is closed and the user or EHR must relaunch the SMART app within the newly opened patient's chart.

I'd like to close this issue because I think that this concern, while real and important, is clearly outside the scope of CDS Hooks. Concerns?

Isaac

**Comment by bvdh** [comment]
Yes, I have. These relate to situations were additional browser windows are opened and need to be closed as the main window changes to a different patient.
In my opionion this is within scope of CDS Hooks. But not in the scope of the v1.0 release. Maybe we can discuss it further in the May meeting in Cologne.

Bas

**Comment by isaacvetter** [comment]
Hey @bvdh - I should've re-pinged you instead of just closing. Happy to talk through this with you, including during the wgm in Cologne. 

My take is that is that FHIRcast answers this need and that it's outside of CDS Hooks' scope. 

Isaac
